support/nginx/peertube

```diff
@@ -1,202 +1,239 @@
+# Minimum Nginx version required:  1.13.0 (released Apr 25, 2017)
+
 # Uncomment in production to redirect HTTP to HTTPS. Leave commented for docker-compose.
 #server {
 #  listen 80;
 #  listen [::]:80;
 #  server_name ${WEBSERVER_HOST};
-#
-#  location /.well-known/acme-challenge/ {
-#    default_type "text/plain";
-#    root /var/www/certbot;
-#  }
-#  location / { return 301 https://$host$request_uri; }
+#  return 301 https://$host$request_uri;
 #}
 
+upstream backend {
+  server ${PEERTUBE_HOST};
+}
+
 server {
   listen 443 ssl http2;
   listen [::]:443 ssl http2;
   server_name ${WEBSERVER_HOST};
 
-  # For example with certbot (you need a certificate to run https)
-  ssl_certificate      /etc/letsencrypt/live/${WEBSERVER_HOST}/fullchain.pem;
-  ssl_certificate_key  /etc/letsencrypt/live/${WEBSERVER_HOST}/privkey.pem;
+  access_log /var/log/nginx/$server_name.access.log; # reduce I/0 with buffer=10m flush=5m
+  error_log  /var/log/nginx/$server_name.error.log;
+
+  ##
+  # Certificates
+  # you need a certificate to run https. see https://letsencrypt.org/
+  ##
+  ssl_certificate     /etc/letsencrypt/live/$server_name/fullchain.pem;
+  ssl_certificate_key /etc/letsencrypt/live/$server_name/privkey.pem;
+
+  location ^~ '/.well-known/acme-challenge' {
+    default_type "text/plain";
+    root /var/www/certbot;
+  }
+
+  ##
+  # Security hardening (as of Nov 15, 2020)
+  ##
 
-  # Security hardening (as of 11/02/2018)
-  ssl_protocols TLSv1.2; # TLSv1.3, TLSv1.2 if nginx >= 1.13.0
+  ssl_protocols             TLSv1.2 TLSv1.3;
   ssl_prefer_server_ciphers on;
-  # Remove ECDHE-RSA-AES256-SHA if you don't want compatibility with Android 4
```

Comment:
So you removed `ECDHE-RSA-AES256-SHA` by default?

Reply:
Yes

Reply:
I mean could you add a comment like *Add ECDHE-RSA-AES256-SHA if you want compatibility with Android 4*?